src/production/production.yml

```diff
@@ -23,6 +23,12 @@ services:
       labels:
         - (( append ))
         - traefik.http.routers.adminer_secure.tls.certresolver=default
+  cloudflared:
+    command: tunnel run
+    environment:
+      TUNNEL_TOKEN: ${TUNNEL_TOKEN}
+    image: cloudflare/cloudflared
+    restart: unless-stopped
```

Comment:
```suggestion
```
This line is not needed for docker stack, this is the default behavior which can't be changed afaik.